Gemfile

```diff
@@ -67,14 +73,16 @@ gem "clamav"
 gem "pdf-reader"
 
 # to generate sitemap for google scholar et al
-gem 'sitemap', github: 'ualbertalib/rails-sitemap'
+gem 'sitemap', git: 'https://github.com/ualbertalib/rails-sitemap.git'
 
 # to fetch noid from fedora for reindex job
 gem 'rest-client'
 
 gem 'noid', '~> 0.8'
 
 group :test do
+  gem 'vcr', require: false
```

Comment:
It's for mocking external API's for testing. In our case EZID. 

So when running your test suite, VCR will intercept any external HTTP request and do the following:

- On the first test run, it will do a real HTTP request to the external service/API (EZID), and record the response/request and cache it in your repo (which can then be version controlled and checked into github). 
- When you run it again, VCR will replay the response from the cached copy instead of doing a real HTTP request again. This makes your test fast (no real HTTP requests are made anymore), deterministic (the test will continue to pass, even if you are offline, or external API goes down) and accurate (the response will contain the same headers and body you get from a real request).  

More info:

https://github.com/vcr/vcr
https://relishapp.com/vcr/vcr/docs